x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -853,3 +853,171 @@ A conscious effort has been made to mask this complexity with sane defaults and
 documentation above but in case you encounter issues while configuring a SAML realm, you can
 look through our {stack-ov}/trb-security-saml.html[SAML troubleshooting documentation] that has
 suggestions and resolutions for common issues.
+
+=== SAML without {kib}
+
+The SAML realm in {es} is designed to allow users to authenticate to {kib} and as
+such, most of the parts of the guide above make the assumption that {kib} is used.
+This section describes how a custom web application could use the relevant SAML
+REST APIs in order to authenticate the users to {es} with SAML.
+
+NOTE: This section assumes that the reader is familiar with the SAML 2.0 standard
+and more specifically with the SAML 2.0 Web Browser Single Sign On profile.
+
+Single sign-on realms such as OpenID Connect and SAML make use of the Token Service in
+{es} and in principle exchange a SAML or OpenID Connect Authentication response for
+an {es} access token and a refresh token. The access token is used as credentials
+for subsequent calls to {es}. The refresh token enables the user to get new {es}
+access tokens after the current one expires.
+
+==== SAML realm
```

Comment:
Likewise, we should have an anchor here too.  Otherwise, it is currently defaulting to "_saml_realm". For example:
```suggestion
[[saml-no-kibana-realm]]
==== SAML realm
```